src/main/java/ua/holotova/hw10/Car.java

```diff
@@ -0,0 +1,56 @@
+/*Создать класс car (будет служить родительским)
+        - с полями: серия, год, цвет, топливо
+        - с методами:
+        I) движение (движется пока есть топливо),
+        II) метод показать статистику
+        - добавить конструктор на 4 поля
+        5) Имплементировать интерфейс в созданных ранее типах машин*/
+
+package ua.holotova.hw10;
+
+public class Car implements Recovery {
+    int series;
+    int year;
+    String color;
+    int fuel;
+
+    public Car(int series, int year, String color, int fuel) {
+        this.series = series;
+        this.year = year;
+        this.color = color;
+        this.fuel = fuel;
+    }
+
+    public void run() {
+        while (fuel > 0) {
+            System.out.println("Car is running");
```

Comment:
fuel level should be decreasing inside cycle, otherwise we have an infinite loop